rust/crd/src/authentication.rs

```diff
@@ -78,6 +81,8 @@ pub enum Error {
         supported: String,
         auth_class_name: String,
     },
+    #[snafu(display("Currently only one CA certificate is supported."))]
+    MultipleCaCertsNotSupported,
```

Comment:
Multiple CA certificates are supported. They are all appended to the ones known by certifi.

```suggestion
```